Gemfile

```diff
@@ -3,7 +3,7 @@ gem 'rspec'
 gem 'serverspec', '~> 2.18.0'
 gem 'ansible_spec'
 gem 'highline'
-
+gem 'rake'
```

Comment:
Could you break this and the `Gemfile.lock` change into a separate commit which explains why we need to add this?